src/test/java/org/entando/entando/web/user/UserControllerUnitTest.java

```diff
@@ -350,7 +337,7 @@ void updateMyPassword_validRequest_shouldUpdatePassword() throws Exception {
 
     @Test
     void updateMyPassword_invalidPassword_shouldReturnBadRequest() throws Exception {
-        UserDetails user = new OAuth2TestUtils.UserBuilder("jack_bauer", "0x24").build();
+        UserDetails user = new OAuth2TestUtils.UserBuilder("jack_bauer", "0x24").grantedToRoleAdmin().build();
```

Comment:
Why "grantedToRoleAdmin"? This API requires only an authenticated user, even without authorizations

Reply:
Thanks. I had to revert that lines but I had forgotten about them.